app/main.py

```diff
@@ -1,3 +1,61 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
```

Comment:
Alternatively, you could put these three “Distance, float, int” types into a separate variable “Variables” and then use it to annotate “other”.
Example:

```
Variables = Distance, int, float

def __add__(self, other: Variables) -> Distance: 
...
```

Otherwise, cool code!